cypress/locators/buttonToggleGroupLocators.js

```diff
@@ -1,21 +1,13 @@
 //knobs locators
 const INPUT_WIDTH_INPUT = '#inputWidth'
```

Comment:
semicolon in the end of variable initialization

Reply:
does not apply this commit, will be fixed in future like we agreed